CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+#Handsontable Contributor Covenant Code of Conduct
```

Comment:
There need to be a space after "#" in order to be formatted correctly. Currently, it displays as a plain text: https://github.com/handsontable/handsontable/blob/06972a09ecec79aee603d839f3b34c46ae6cde50/CODE_OF_CONDUCT.md